.github/ISSUE_TEMPLATE/work-item-issue.md

```diff
@@ -0,0 +1,48 @@
+# Work Item Issue Template
+description:
+title: "[Work_Item ]"
+labels: ["work item"]
+assignees: [shawnalpay, jpradocueva]
+## 1. **Problem Statement** *
+Describe the problem, issue, use case, or opportunity that this work item addresses.
+- **What is the problem?**: Explain the context and why it needs resolution.
+- **Impact**: Describe how the problem affects users, systems, or the project.
```

Comment:
I doubt most people will really be able to give an effective impact statement past their experience, so rolling this up with line 8 makes more sense to me.  Either add "impact" into line 8 or treat "why it needs resolution" as enveloping impact.

Reply:
Hmm, interesting -- I think it makes sense to carve out separate spaces where people can describe the context of the issue alongside how it affects them.